jest.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
-  testEnvironment: "jsdom",
+  testEnvironment: 'jsdom',
   moduleNameMapper: {
-    ".(css|less|scss)$": "identity-obj-proxy",
+    '.(css|less|scss)$': 'identity-obj-proxy',
   },
+  modulePathIgnorePatterns: ['node_modules', 'testOutput.json'],
```

Comment:
Exclude testOutput file from being watched, otherwise tests run in an infinite loop in watch mode.